src/twisted/internet/test/test_process.py

```diff
@@ -836,10 +836,10 @@ def test_shebang(self):
         evaluate the script.
         """
         shebangOutput = b"this is the shebang output"
-
+        executable = getattr(sys, "_base_executable", pyExe.decode("ascii"))
```

Comment:
I know that we have the git commit message...but maybe a comment can help to understand what is going on there, without having to run `git blame`
```suggestion
        # On Windows with Python 3.13, sys.executable seems to result in some weird output,
        # so try the original executable.
        executable = getattr(sys, "_base_executable", pyExe.decode("ascii"))
```